cmd/sfncli/runner_test.go

```diff
@@ -0,0 +1,270 @@
+package main
+
+import (
+	"context"
+	"os"
+	"path"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/Clever/sfncli/gen-go/mocksfn"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sfn"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+)
+
+const mockTaskToken = "taskToken"
+
+const emptyTaskInput = "{}"
+
+var testScriptsDir = os.Getenv("GOPATH") + "/src/github.com/Clever/sfncli/cmd/sfncli/test_scripts"
```

Comment:
This might cause problems if multiple directories are specified in the `GOPATH`

Reply:
yeah I couldn't think of anything better...

Reply:
I *think* go does magic to make relative paths work in unit tests.  IIRC, we take advantage of it for app log-routing unit tests.

Reply:
63f702c